### DISCUSSION TITLE
A way to check only FN (False Negative) and FP (False Positive) separately in the coco result?

### DISCUSSION THREAD

**Discussion by YoungjaeDev** [obss/sahi#680] [Ideas]
Hello
I checked that the mAP comes out when I do SAHI EVALUATE (GOOD! :+1:  ), but is there a separate API designed to check(save) only the images where FN (False Negative) and FP (False Positive) occurred?

Or is it right to implement it myself?

THANK U

**Comment by devrimcavusoglu** [comment]
Hi thanks for the issue, this is not on the roadmap, you can implement yourself, or if you're willing to contribute, feel free to open a pull request :)
